sycl/test-e2e/SubGroup/scan_spirv13.cpp

```diff
@@ -14,6 +14,8 @@
 // This test verifies the correct work of SPIR-V 1.3 exclusive_scan() and
 // inclusive_scan() algoriths used with the operation MUL, bitwise OR, XOR, AND.
 
+#error FIXME: Test times-out
```

Comment:
Another way to fix this is to change XFAIL on line 9 to UNSUPPORTED, which way do you think is better?

Reply:
`GroupAlgorithm/SYCL2020/reduce_over_group_size.cpp` too.

Reply:
> `GroupAlgorithm/SYCL2020/reduce_over_group_size.cpp` too.

I had to follow a different a approach for that one, but it's `XFAIL`ing now

Reply:
You may not need to follow a different approach for that one, as it is marked XFAIL in the xfail_test.txt. And XFAIL test cases are not run on GPU.

Reply:
Let's try

Reply:
Looks like the `#error` way doesn't solve the time out issue:
```
Timed Out Tests (3):
  SYCL :: SubGroup/scan_spirv13.cpp
  SYCL :: USM/copy2d.cpp
  SYCL :: USM/memcpy2d.cpp
```

Reply:
`cgeist` has incorrect behavior here. https://github.com/intel/llvm/pull/9306 fixes that and should be merged before this